database/helpers/insertionHelpers.js

```diff
@@ -54,14 +57,40 @@ module.exports.addBookingPrep = (obj) => {
 
 module.exports.addBookingDetailBulk = (bookingArray, nightlyArray) => {
   if (bookingArray.length === 0) return [];
+  const startTime = new Date();
+
   const bookingText = `INSERT INTO listings (listing_id, market, neighbourhood, room_type, review_scores_rating) 
     VALUES ${bookingArray.join(', ')} 
     ON CONFLICT (listing_id) 
     DO NOTHING`;
   const nightlyText = `INSERT INTO booked_nights (listing_id, booked_at, price, search_id) 
     VALUES ${nightlyArray.join(', ')}`;
-
-  return client.query(bookingText).then(client.query(nightlyText)).catch(console.log);
+  let startNightlyTime;
+  return client.query(bookingText)
+    .then((data) => {
+      logger.log({
+        queryName: 'addBookingDetailBulk-Booking',
+        database: 'postgres',
+        level: 'info',
+        type: 'log',
+        rows: data.rows,
+        elapsed: new Date() - startTime,
+      });
+      startNightlyTime = new Date();
+      return client.query(nightlyText);
+    })
+    .then((nightlyData) => {
+      logger.log({
+        queryName: 'addBookingDetailBulk-Nightly',
+        database: 'postgres',
```

Comment:
The repetition with database, level, and type suggests to me a base logging parameter object can be defined for the module and extended when passed into the logger.log method. This could be used to remove the bulk that this new logging code is adding.

Reply:
The only two extendable parameters are these two:     
   level: 'info',
        type: 'log',
I could refactor it out into an object but for now I will keep it as is until more constant parameters show up.